### PR TITLE
fix: chart xAxis; add smooth line

### PR DESCRIPTION
**Pull request by ZonaHex**
Make chart xAxis encode name to be the first timestamp name; Add line(smooth) into chart type options;
**Do not merge yet.**
Trying to fix delete tab and chart.

### CODE REVIEW

File: src/hooks/data-explorer.ts [hunk 1]
```diff
@@ -1,9 +1,10 @@
 const columns = ref<any>([])
 
 // todo: change init code
-const code = ref('select * from scripts')
+const code = ref('select * from system_metrics')
```
Comment (sunng87) [inline src/hooks/data-explorer.ts:4]: this table is not built-in, it's just from our docs. It's not recommended to make it default.

Comment (ZonaHex) [reply src/hooks/data-explorer.ts:4]:
```suggestion
const code = ref('select * from scripts')
```

**Comment by ZonaHex** [comment]
Ready for review now. Will fix delete tab in other pr.

**Review by alili — APPROVED** [review]
The branch code has been verified

**Pull request merged**